src/test/java/seedu/address/model/person/TutorialTest.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.model.person;
+
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.address.testutil.Assert.assertThrows;
+
+import org.junit.jupiter.api.Test;
+
+public class TutorialTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new Tutorial(null));
+    }
+
+    @Test
+    public void constructor_invalidTutorial_throwsIllegalArgumentException() {
+        String invalidTutorial = "A";
+        assertThrows(IllegalArgumentException.class, () -> new Tutorial(invalidTutorial));
+    }
+
```

Comment:
This tests seems to have failed the java ci, there might have been parser issues?

Reply:
Thanks for pointing that out, it was a Tutorial `VALIDATION_REGEX` issue, issue fixed.